ledger/eval.go

```diff
@@ -1171,22 +1163,35 @@ func eval(ctx context.Context, l ledgerForEvaluator, blk bookkeeping.Block, vali
 
 	}
 
-	for _, txgroup := range paysetgroups {
+	base := eval.state.lookupParent.(*roundCowBase)
+
+	for {
 		select {
+		case txgroup, ok := <-paysetgroupsCh:
+			if !ok {
+				break
```

Comment:
```suggestion
outer:
	for {
		select {
		case txgroup, ok := <-paysetgroupsCh:
			if !ok {
				break outer
```

Reply:
It's the first time I'm seeing a use for the `break label`, but I don't see why not to use it.

Reply:
The performance gains are about 20% faster account loading, which is notable when you have full blocks.
When looking on the overall `eval()` gains, it's roughly 10% of the total time.

The key here is that we want to perform all the account and sql operation done in parallel, and (potentially) ahead of transaction processing. We might want to include the asset loading here as well, but this could wait.

Reply:
yep, in couple places we have goto instead of `break label` or `continue label` and there goto is fine. Here I think the code will be much simpler with this construct.